srcpkgs/drawterm-9front/template

```diff
@@ -0,0 +1,48 @@
+# Template file for 'drawterm-9front'
+pkgname=drawterm-9front
+version=r393.cdbacd80f61a
+revision=1
+hostmakedepends="mercurial"
+makedepends="libX11-devel libXt-devel"
+short_desc="Drawterm fork by 9front project"
+maintainer="Juan Cuzmar <juan.cuzmar.s@gmail.com>"
+license="LPL-1.02"
+homepage="http://drawterm.9front.org/"
+CFLAGS="-fno-strict-aliasing"
+
+if [ "${XBPS_TARGET_MACHINE}" = armv5tel ]; then
+	CFLAGS+=" -DARMv5"
+fi
+
+do_fetch() {
+	hg clone https://code.9front.org/hg/drawterm ${wrksrc}
+}
+
+do_configure() {
+	local _arch;
+
+	case "$XBPS_TARGET_MACHINE" in
+		arm*) _arch=arm ;;
+		arm64*) _arch=arm64 ;;
+		i686*) _arch=386 ;;
+		mips*) _arch=mips ;;
+		x86_64*) _arch=amd64 ;;
+		ppc*) _arch=power ;;
```

Comment:
Does this work for ppc and ppc64?

Reply:
Only works for [ppc](https://code.9front.org/hg/drawterm/file/cdbacd80f61a/posix-power)

Reply:
We can test it out first, but you might need to specify that it's broken for ppc64.

Reply:
I ask to some people who use drawterm daily and they told that doesn't work for ppc64.

Reply:
In that case you need to makr ppc64 as broken. And throw an error if the case statement doesn't match anything.

Reply:
just do the same thing with tas.c as regular drawterm does